### ISSUE TITLE
Cannot create a ref to variable in struct

### ISSUE THREAD

**Issue by BenjaminHalko** [YoYoGames/GameMaker-Bugs#1730]
### Description
You cannot create a debug ref from a struct, despite in stating in the manual that you can.

### Steps To Reproduce
Attempt to create a ref from a struct:
```
the_struct = {a: "text", b: 485};
ref_to_struct = ref_create(self, "the_struct");
ref_to_struct_var = ref_create(ref_to_struct, "a");
```

[135c15e2-53e3-4e7d-a614-5b7c289d779b](https://api.gamemaker.io/api/github/downloads/135c15e2-53e3-4e7d-a614-5b7c289d779b)

**Comment by BenjaminHalko** [comment]
I receive this error:
```
ref_create argument 2 incorrect type (string) expecting a Number (YYGI32)
at gml_Object_oDebug_Create_0 (line 17) - ref_to_struct_var = ref_create(ref_to_struct, "a");
```

**Comment by BenjaminHalko** [comment]
This code was taken straight out of the manual.

**Comment by rwkay** [comment]
Fixe merged into 2023.11

**Comment by Emc1923** [comment]
Verified fixed internally in   IDE v2023.1100.0.415  Runtime v2023.1100.0.436
